lib/ui/form/widget/form_survey_answer_dropdown.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/model/question_model.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+class FormSurveyAnswerDropdown extends StatefulWidget {
+  final QuestionModel question;
+
+  const FormSurveyAnswerDropdown({super.key, required this.question});
+
+  @override
+  State<FormSurveyAnswerDropdown> createState() =>
+      _FormSurveyAnswerDropdownState();
+}
+
+class _FormSurveyAnswerDropdownState extends State<FormSurveyAnswerDropdown> {
+  late List<String> _pickerOptions = [];
+
+  int _selectedIndex = 1;
+  late FixedExtentScrollController _scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+  }
```

Comment:
I don't think we need to override this function

Reply:
Now we need this as initially we select the middle element.